docs/deployment/index.rst

```diff
@@ -0,0 +1,266 @@
+.. _Deployment:
+
+FLINT.Cloud Deployment
+####################
+
+This section guides you through the process of deploying FLINT.Cloud to
+your own server, through a public cloud service provider. We are using
+Google Cloud Platform (GCP) and Azure, as of now, to deploy FLINT.Cloud
+APIs to be consumed for your FLINT-specific applications, like FLINT-UI.
+This guide assumes that you have some preliminary understanding of the
+concepts of GCP and Azure and have deployed an application before.
+
+Google Cloud Platform
+---------------------
+
+To deploy FLINT.Cloud to Google Cloud Platform, we provide a
+production-grade setup with a Layered Architecture Setup on top of the
+Google Cloud. This setup includes the usage of `Terraform`_, an
+Infrastructure-as-a-Code tool, to deploy the infrastructure and the
+application to be used.
+
+To deploy the FLINT.Cloud to Google Cloud Platform, follow these steps:
+
+1. Create a GCP service account with project owner permissions in your
```

Comment:
Dont recommend giving project owner permission for service accounts. Give the permission required to deploy the infrastructure such as Compute admin etc.